### DISCUSSION TITLE
"brace style" of "if statement"

### DISCUSSION THREAD

**Discussion by usualoma** [honojs/hono#2108] [General]
Several PRs have recently added changes as refactorings that apply the following rules

https://eslint.org/docs/latest/rules/curly#multi

If I may say my preference, I like `#all`, but leaving that aside, the "brace style" of "if statement" is a problem in modern JavaScript that should be solved with a linter, and a review in the PR I don't think it is a problem that should be solved at the cost of a PR review.

If the project decides on a rule ( `#multi` or `#all` or …), it should be left to a linter, and if not, I think PR should be prohibited from changing "if statements".

**Comment by yusukebe** [comment]
That's really true. And I also like `all` style.

I've updated the lint rules. It will be introduced into the core soon!

https://github.com/honojs/middleware/pull/365
